components/forms/UserPasswordForm.js

```diff
@@ -0,0 +1,62 @@
+import { useMutation } from '@tanstack/react-query';
+import { getAuthToken } from 'lib/client';
+import { useApi } from 'next-basics';
+import { useRef } from 'react';
+import { SubmitButton, Form, FormButtons, FormInput, PasswordField } from 'react-basics';
+import styles from './UserForm.module.css';
+
+export default function UserPasswordForm({ data, onSave }) {
```

Comment:
Another password form??

Reply:
If all these password forms do the same thing, we can use one with different props.